Sources/PersistentCollections/PersistentDictionary._Node.swift

```diff
@@ -750,21 +757,11 @@ extension PersistentDictionary._Node {
   func _copyAndSetValue(
     _ isUnique: Bool, _ bucket: _Bucket, _ newValue: Value
   ) -> _Node {
-    let src: _Node = self
-    let dst: _Node
-
-    if isUnique {
-      dst = src
-    } else {
-      dst = src.copy()
-    }
-
+    let dst = isUnique ? self : self.copy()
     let offset = itemMap.offset(of: bucket)
-
     dst.itemBaseAddress[offset].value = newValue
-
-    assert(src.invariant)
-    assert(dst.invariant)
+    _invariantCheck()
```

Comment:
To be consistent with edits below:

```suggestion
    self._invariantCheck()
```